### DISCUSSION TITLE
PurpleRestore Icon

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1349] [Icon request] [answered]
PurpleRestore is a apple internal tool used to restore iPhone's 
Its meant to restore phones to internal beta firmware how ever it can be used to install normal firmware on retail phones. 

Here is the link to the original icon https://www.theiphonewiki.com/w/images/b/b3/PurpleRestore3_logo.png
Here is a link to the wiki page about it for those interested. https://www.theiphonewiki.com/wiki/PurpleRestore

**Comment by 7kayoh** [comment, marked as answer]
[PurpleRestore.png.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7877274/PurpleRestore.png.zip)

Preview with background: 
<img width="479" alt="CleanShot Sketch NL5E0amq" src="https://user-images.githubusercontent.com/40730127/149666769-204e1e70-93bd-402d-b213-30b45c6e1fec.png">

**Reply by ghost** [reply]
Thanks!

**Reply by ghost** [reply]
oof. i just realized that it changes the icon back to default because it has a progress indicator in the icon

**Reply by ghost** [reply]
well thanks anyways!

**Reply by ghost** [reply]
wait i just found something out while messing around with it. apparently it adds the progress bar on its own? im gonna edit out the existing progress bar and see if it works

**Reply by ghost** [reply]
well i think the position of the phone needs to be adjusted
<img width="51" alt="Screen Shot 2022-01-17 at 5 56 04 PM" src="https://user-images.githubusercontent.com/96893300/149852727-0918fd46-2f30-4a21-9691-612ffd2616b6.png">

**Reply by 7kayoh** [reply]
The original icon has its progress bar tilted along with the iDevice, do you think it's possible to have the progress bar tilted?

I can try exporting it into an ICNS which contains the progress bar moving from 0 to 100

**Reply by ghost** [reply]
> The original icon has its progress bar tilted along with the iDevice, do you think it's possible to have the progress bar tilted?
> 
> I can try exporting it into an ICNS which contains 
the progress bar moving from 0 to 100

Maybe? it appears the application adds the bar in but also it might have the progress bar as icons? if you want to see the og files here you go
[Archive.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7885673/Archive.zip) (note this is a zip of the icons not the actual purplerestore app)

**Reply by ghost** [reply]
actually i think upon inspection the app makes the bar. but yet there is a icns for a icon used when a restore is going on?
